kyuubi-server/pom.xml

```diff
@@ -251,6 +257,11 @@
             <artifactId>trino-client</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-jdbc</artifactId>
+        </dependency>
```

Comment:
do we need import `trino-jdbc` ? I see the dependency list changed.

Reply:
jdbc tests were written on the server before，them were moved to integration-test and i will remove it